cli/src/clients/datafusion_http_client.rs

```diff
@@ -57,17 +61,30 @@ impl DataFusionHttpClient {
         Ok(Self {
             inner: raw_client,
             base_url: env.datafusion_http_base_url.clone(),
+            bearer_token: env.bearer_token.clone(),
+            request_timeout: env.request_timeout.clone(),
         })
     }
 
+    /// Prepare a request builder for a DataFusion request.
+    fn prepare(&self, path: Url) -> reqwest::RequestBuilder {
+        let request_builder = self
+            .inner
+            .request(reqwest::Method::POST, path)
+            .timeout(self.request_timeout);
```

Comment:
Given that df queries might take longer than the default 10 seconds, one might consider the `connect_timeout` as a reasonable timeout strategy for now. 

At least, that would eliminate indefinitely waiting on unreachable hosts, what do you think?

Reply:
That makes sense, let's switch to `connect_timeout` instead. It will catch obvious misconfigurations without getting in the way of long-running queries. Thanks, update coming up!